code/modules/reagents/newchem/newchem_procs.dm

```diff
@@ -139,22 +132,22 @@ datum/reagent/proc/overdose_start(var/mob/living/M as mob)
 
 datum/reagent/proc/addiction_act_stage1(var/mob/living/M as mob)
 	if(prob(30))
-		M << "<span class = 'notice'>You feel like some [name] right about now.</span>"
+		M << "<span class='notice'>You feel like some [name] right about now.</span>"
 	return
 
 datum/reagent/proc/addiction_act_stage2(var/mob/living/M as mob)
 	if(prob(30))
-		M << "<span class = 'notice'>You feel like you need [name]. You just can't get enough.</span>"
+		M << "<span class='notice'>You feel like you need [name]. You just can't get enough.</span>"
 	return
 
 datum/reagent/proc/addiction_act_stage3(var/mob/living/M as mob)
 	if(prob(30))
-		M << "<span class = 'danger'>You have an intense craving for [name].</span>"
+		M << "<span class='danger'>You have an intense craving for [name].</span>"
 	return
 
 datum/reagent/proc/addiction_act_stage4(var/mob/living/M as mob)
 	if(prob(30))
-		M << "<span class = 'danger'>You're not feeling good at all! You really need some [name].</span>"
+		M << "<span class='danger'>You're not feeling good at all! You really need some [name].</span>"
```

Comment:
Because some individuals, who shall not e named, like to nitpick and gets PRs held up for spacing issues :smile: